File: ParkingApi/Models/ParkingSession.cs
namespace ParkingApi.Models
{
    public class ParkingSession
    {
        public int Id { get; set; }
        public required string Plate { get; set; }
        public required DateTimeOffset EntryTime { get; set; }
        public required decimal RatePerMinute { get; set; }
        public DateTimeOffset? ExitTime { get; set; }
        public int MinutesCharged {  get; set; }
        public decimal Subtotal { get; set; }
        public decimal Taxes { get; set; }
        public decimal Total { get; set; }
    }
}

File: ParkingApp/src/types/ParkingSession.ts
export interface ParkingSession {
	id: number;
	plate: string;
	entryTime: string; // ISO string format from DateTimeOffset
	ratePerMinute: number;
	exitTime?: string | null; // nullable DateTimeOffset
	minutesCharged: number;
	subtotal: number;
	taxes: number;
	total: number;
}
